src/TouristInfoCards.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+const TouristInfoCards = () => {
+  return (
+    <div className="container-tourist">
+      <div className="card">
```

Comment:
To make this even better you could extra each of these `card`s into their own component. 

In React, it is always better to create small, manageable components to save time when writing the same code again and again 😊